.github/workflows/stale.yaml

```diff
@@ -0,0 +1,46 @@
+name: Mark and Close Stale Issues and PRs
```

Comment:
Is this a best practice?

Reply:
thanks for your comment, do you mean it only should mark (+close) issues and not prs?